cmd/release/main.go

```diff
@@ -314,25 +318,33 @@ type git struct {
 }
 
 func (g git) tag(version string) error {
-	// if the tag exists then return nil; if it doesn't create it
-	cmd := exec.Command("git", "rev-parse", "--quiet", "--verify", version)
-	_, err := cmd.CombinedOutput()
+	//TODO: this could probably use some work
+	if oldRev, err := g.tagRevision(version); err != nil {
+		g.logger.Info("non critical error, failed to get tag revision: %v", err)
+	} else {
+		g.logger.Infof("You are forcing the update of a tag reference from %v to the current commit", oldRev)
```

Comment:
We should probably prompt before attempting to force overwriting of an existing tag.